tasks/copy-template/README.md

```diff
@@ -2,8 +2,7 @@
 
 #### Task is deprecated and will be removed in future versions.
 
-This task copies a template.
-A bundle of predefined templates to use can be found in [Common Templates](https://github.com/kubevirt/common-templates) project.
+This task copies an KubeVirt Virtual Machine template. This task works only with Common Templates and on OKD / OpenShift. A bundle of predefined templates to use can be found in [Common Templates](https://github.com/kubevirt/common-templates) project.
```

Comment:
Is it really limited to common-templates. Wouldn't it work with any `Template`?

Reply:
Correct, copy template task can work with any template